reporter/config.go

```diff
@@ -59,6 +59,8 @@ type SymbolUploaderConfig struct {
 	UploadGoPCLnTab bool
 	// BatchSymbolQueries defines whether the uploader should batch symbol queries.
 	SymbolQueryInterval time.Duration
+	// Force disabling debug section compression whatever objcopy support.
```

Comment:
```suggestion
	// DisableDebugSectionCompression  defines whether the uploader should disable debug section compression.
```
What do you think of changing the comment to something like this for it to remain cohesive with the present ones?

Reply:
Fixed